lib/rules/property-no-deprecated/__tests__/index.mjs

```diff
@@ -0,0 +1,123 @@
+import rule from '../index.mjs';
+import { stripIndent } from 'common-tags';
+
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: [true],
+	fix: true,
+	computeEditInfo: true,
+
+	accept: [
+		{
+			code: 'a { clip-path: auto; }',
+		},
+	],
+
+	reject: [
+		{
+			code: 'a { clip: auto; }',
```

Comment:
minor

Please don't use `auto` use `rect()`.
`auto` is the default for `clip` but it will be ignored by `clip-path`, which is fine but not worth testing.
i.e. the test needs to make sense

Reply:
Fixed